.github/workflows/e2e-cross-version-for-breaking-changes.yml

```diff
@@ -50,9 +88,57 @@ jobs:
             ${{ inputs.fe-ref }} ${{ inputs.be-ref }} \
             build
 
+      - name: Upload Uberjar
+        uses: actions/upload-artifact@v4
+        with:
+          name: metabase-jar-cross-version
+          path: .tmp/metabase-be/target/uberjar/metabase.jar
```

Comment:
I wonder who puts this uberjar in `.tmp/metabase-be` for us? I haven't seen this directory before, so I guess it must be one of the GitHub actions we have.

Reply:
it makes sense now, thanks!